postgraphile/website/postgraphile/make-extend-schema-plugin.md

```diff
@@ -606,6 +606,36 @@ export const MyForeignExchangePlugin = makeExtendSchemaPlugin((build) => {
 });
 ```
 
+## Returning Connection
+When returning a connection in `makeExtendSchemaPlugin`, you must wrap the return value with `connection(...)`.  
```

Comment:
```suggestion
When returning a connection in `makeExtendSchemaPlugin`, you must be sure that your plan resolver yields a `connection(...)` step; failure to do this may result in errors such as `$connection.getSubplan is not a function`.
```